game.py

```diff
@@ -361,7 +359,7 @@ def _my_strategy_import(self, f, arg):
             self._set_label('Python overflow error: {}'.format(e))
         except TypeError as e:
             self._set_label('Python type error: {}'.format(e))
-        except:
+        except BaseException:
```

Comment:
We should avoid _BaseException_ if there is a specific exception we should be handling instead.  Is there a specific exception expected?